blocks/extra-large-manual-promo-block/features/extra-large-manual-promo/default.jsx

```diff
@@ -68,10 +68,10 @@ const ExtraLargeManualPromo = ({ customFields }) => {
 
 	const alt = headline || description || null;
 	const imageParams =
-		imageId && imageURL
+		imageURL && resizedAuth
 			? {
 					ansImage: {
-						_id: resizedImage && imageId,
+						_id: resizedImage ? imageId : "",
 						url: imageURL,
 						auth: resizedAuth || {},
```

Comment:
@blakeganderson , if we're checking for `resizedAuth` in the above condition, then this should always have a value. So we shouldn't need the `|| {}` part anymore.

Reply:
removed it from the other 3 manual promos 👍